--- /etc/influxdb/influxdb.conf.origin	2021-07-30 11:27:28.763893985 +0800
+++ /etc/influxdb/influxdb.conf	2022-03-04 08:39:17.807985271 +0800
@@ -10,6 +10,7 @@
 # usage data. No data from user databases is ever transmitted.
 # Change this option to true to disable reporting.
 # reporting-disabled = false
+reporting-disabled = true
 
 # Bind address to use for the RPC service for backup and restore.
 # bind-address = "127.0.0.1:8088"
@@ -23,7 +24,7 @@
 
 [meta]
   # Where the metadata/raft database is stored
-  dir = "/var/lib/influxdb/meta"
+  dir = "BARRELE_INFLUXDB_PATH/meta"
 
   # Automatically create a default retention policy when creating a database.
   # retention-autocreate = true
@@ -42,10 +43,10 @@
 
 [data]
   # The directory where the TSM storage engine stores TSM files.
-  dir = "/var/lib/influxdb/data"
+  dir = "BARRELE_INFLUXDB_PATH/data"
 
   # The directory where the TSM storage engine stores WAL files.
-  wal-dir = "/var/lib/influxdb/wal"
+  wal-dir = "BARRELE_INFLUXDB_PATH/wal"
 
   # The amount of time that a write will wait before fsyncing.  A duration
   # greater than 0 can be used to batch up multiple fsync calls.  This is useful for slower
@@ -57,7 +58,7 @@
   # The type of shard index to use for new shards.  The default is an in-memory index that is
   # recreated at startup.  A value of "tsi1" will use a disk based index that supports higher
   # cardinality datasets.
-  # index-version = "inmem"
+  index-version = "tsi1"
 
   # Trace logging provides more verbose output around the tsm engine. Turning
   # this on can provide more useful output for debugging tsm engine issues.
@@ -120,7 +121,7 @@
   # The maximum series allowed per database before writes are dropped.  This limit can prevent
   # high cardinality issues at the database level.  This limit can be disabled by setting it to
   # 0.
-  # max-series-per-database = 1000000
+  max-series-per-database = 0
 
   # The maximum number of tag values per tag that are allowed before writes are dropped.  This limit
   # can prevent high cardinality tag values from being written to a measurement.  This limit can be
@@ -364,7 +365,7 @@
   # Determines which level of logs will be emitted. The available levels
   # are error, warn, info, and debug. Logs that are equal to or above the
   # specified level will be emitted.
-  # level = "info"
+  level = "warn"
 
   # Suppresses the logo output that is printed when the program is started.
   # The logo is always suppressed if STDOUT is not a TTY.
@@ -493,9 +494,9 @@
 ###
 
 [[opentsdb]]
-  # enabled = false
-  # bind-address = ":4242"
-  # database = "opentsdb"
+  enabled = true
+  bind-address = ":4242"
+  database = "BARRELE_INFLUXDB_DATABASE_NAME"
   # retention-policy = ""
   # consistency-level = "one"
   # tls-enabled = false
